node/kagome_full/kagome_options.hpp

```diff
@@ -24,6 +25,7 @@ namespace kagome::options {
     INVALID_CONFIG_FILE,           // failed to open configuration file
     CANNOT_OPEN_FILE,              // cannot open file
     LEVELDB_PATH_IS_NOT_DIR,       // leveldb path must be directory
+    INVALID_ENDPOINT,
```

Comment:
Comment it just to be consistent with other constants. Or remove comments from them, they don't seem to be much more informative than constants names.

Reply:
I think is better to remove redundant comments.